test/integration/submatrix/sqldb.js

```diff
@@ -135,7 +244,81 @@ azuresqldb = {
     'password': '<string>',
     'uri': '<string>'
   },
-  e2e: true
+  e2e: true,
+  envVars: envVars2,
+  preProvision: function(callback) {
+    var opParams = {
+      'organization_guid': uuid.v4(),
+      'plan_id': '3819fdfa-0aaa-11e6-86f4-000d3a002ed5',
+      'service_id': 'fb9bc99e-0aa9-11e6-8a8a-000d3a002ed5',
+      'space_guid': uuid.v4(),
+      'privilege': {
+        'sqldb': {
+          'allowToCreateSqlServer': true
+        }
+      },
+      'accountPool': { 'sqldb': {} },
+      'parameters': {
+        'resourceGroup': resourceGroupName2,
+        'location': location,
+        'sqlServerName': sqlServerName2,
+        'sqlServerParameters': {
+          'allowSqlServerFirewallRules': [
+            {
+              'ruleName': 'test',
+              'startIpAddress': '0.0.0.0',
+              'endIpAddress': '255.255.255.255'
+            }
+          ],
+          'properties': {
+            'administratorLogin': 'azureuser',
+            'administratorLoginPassword': 'c1oudc0w!@#'
+          }
+        },
+        'sqldbName': sqldbNamePre2,
+        'sqldbParameters': {
+          'properties': {
+            'collation': 'SQL_Latin1_General_CP1_CI_AS'
+          }
+        }
+      },
+      'azure': common.getConfigurations().azure
+    };
+    sqldb.provision(opParams, function(err) {
```

Comment:
Different from sqldb failover group module, it is a "circle in proving" here. But it seems no better choice to. So it's okay to me.